spec/controllers/email_preferences_controller_spec.rb

```diff
@@ -2,14 +2,27 @@
 
 describe Users::EmailPreferencesController do
   describe "edit" do
-    let(:user) { create (:user) }
+    let(:user) { create :user, angular_ui_enabled: true }
 
-    it 'renders the beta interface when an unsubscribe token is present' do
+    it 'renders the beta interface for betea user when an unsubscribe token is present' do
```

Comment:
Can we ditch the beta page altogether? It's totally different to the angular one. I'd be keen to work out how we could make the angular page available to signed out users with unsubscribe tokens.